api/src/main/java/org/openmrs/validator/PersonValidator.java

```diff
@@ -48,7 +48,7 @@ public boolean supports(Class<?> clazz) {
 	/**
 	 * @see org.springframework.validation.Validator#validate(java.lang.Object,
 	 *      org.springframework.validation.Errors)
-	 * <strong>Should</strong> fail validation if birthdate makes patient older that 120 years old
+	 * <strong>Should</strong> fail validation if birthdate makes patient older that 140 years old
```

Comment:
thanks @Ndacyayisenga-droid  ,change that to than

Reply:
Thanks @HerbertYiga  done